release-tools/build.make

```diff
@@ -52,6 +53,7 @@ IMAGE_TAGS+=$(shell tagged="$$(git describe --tags --match='v*' --abbrev=0)"; if
 
 # Images are named after the command contained in them.
 IMAGE_NAME=$(REGISTRY_NAME)/$*
+MULTIARCH_IMAGE_NAME=gcr.io/k8s-staging-csi/$*
```

Comment:
I don't think we need a separate `MULTIARCH_IMAGE_NAME`. Just use `IMAGE_NAME` and make sure that `cloudbuild.yaml` overrides it with `make push-multiarch REGISTRY_NAME=gcr.io/k8s-staging-csi`.

Reply:
Removed `MULTIARCH_IMAGE_NAME` extra variable. Added `REGISTRY_CI` env variable in cloudbuild to override `REGISTRY_NAME` value in makefile